ClientApp/KachnaOnline/src/app/navigation-bar/loading-spinner/loading-spinner.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="spinner-border" role="status" *ngIf="loaderService.loading | async">
+  <span class="sr-only">Načítám...</span>
+</div>
```

Comment:
I am wondering, though, whether we could use this spinner somewhere else in the application. If yes, we should move the spinner to shared components folder with modifiable text option, `načítám...` being the default one.

Reply:
We decided to leave it be for now as it is.